src/author/author.repository.ts

```diff
@@ -0,0 +1,71 @@
+import { Injectable } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
+import { Author } from "./entities/author.entity";
+import { Repository } from "typeorm";
+import { CreateAuthorDto } from "./dtos/create-author.dto";
+import { UpdateAuthorDto } from "./dtos/update-author.dto";
+
+@Injectable()
+export class AuthorRepository {
+    constructor (@InjectRepository(Author)
+                    private authorRepository: Repository<Author>) {}
+
+   async create(createAuthorDto :  CreateAuthorDto){
+        const author = await this.authorRepository
+        .createQueryBuilder()
+        .insert()
+        .into(Author)
```

Comment:
When you describe the repository in the controller it is no longer necessary to use into